eng/automation/generate_utils.py

```diff
@@ -221,7 +221,7 @@ def compare_with_maven_package(
             if not os.path.exists(new_jar):
                 raise Exception("Cannot found built jar in {0}".format(new_jar))
             breaking, changelog, breaking_changes = generate_changelog_and_breaking_change(sdk_root, old_jar, new_jar)
-            if changelog is not None:
+            if changelog is not None and write_changelog:
```

Comment:
Maybe we can always write_changelog? (is the False for automation from specs?)

Reply:
> is the False for automation from specs repo?

Yes. 
Seems update_changelog will overwrite everything for the first version. Indeed we can always write changelog.

Reply:
updated